client/packages/invoices/src/OutboundShipment/DetailView/columns.tsx

```diff
@@ -170,6 +172,53 @@ export const useOutboundColumns = () => {
           }
         },
       },
+      {
+        id: 'volume',
+        header: t('label.volume'),
+        size: 100,
+        columnType: ColumnType.Number,
+        accessorFn: rowData => {
+          if ('subRows' in rowData) {
+            return (rowData.subRows ?? []).reduce(
+              (sum, batch) =>
+                sum +
+                (batch.stockLine?.volumePerPack ?? 0) * batch.numberOfPacks,
+              0
+            );
+          } else {
+            return (
+              (rowData.stockLine?.volumePerPack ?? 0) * rowData.numberOfPacks
+            );
+          }
+        },
+        Footer: ({ table }) => {
```

Comment:
Decided to use this instead of hiding it in side panel... we can probably move out total to here as well and keep total after tax in side panel 👀 More visibility this way...